recipes/rabpro/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "rabpro" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/veinsoftheearth/rabpro/archive/v{{ version }}.tar.gz
+  sha256: 5cac062de2d715eb57d5f839afb9a18b084df23a3a5479ef2bfc9d95d81d17c8
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.  
+
+build:
+  noarch: python
+  number: 0  
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:  
+  host:
+    - pip
+    - python >=3.7,<3.10
+    
+  
+  run:
+    - python >=3.7,<3.10
```

```suggestion
    - python >=3.7
```
Looks like upstream only has a minimum python version; are you sure that this package isn't forward compatible? If you are afraid of python=4, you can use pin_compatible to the major version.

https://github.com/VeinsOfTheEarth/rabpro/blob/main/setup.py

---

If `fiona` is incompatible with python 3.11, then that is something that the `fiona` recipe will enforce. This recipe should only describe the requirements of rabpro.